src/pages/index.js

```diff
@@ -67,23 +69,23 @@ const IndexPage = ({ data }) => {
           <Title
             fontSize={[3, 'desktop.3']}
             lineHeight={2}
-            borderWidth={0}
+            border='none'
             mt={0}
           >
             React & React Native
           </Title>
-          <Box width={[1, 2/3]}>
-            <P textAlign="center">
+          <Box width={['100%', 2/3]}>
+            <Text textAlign="center" fontSize={[1, 'desktop.1']} mb='2'>
```

Comment:
we are spilling font size like props everywhere, we should control font size to theme.json . and only one offs need to be themed